cypress/support/e2e.ts

```diff
@@ -30,3 +30,7 @@ declare global {
     }
   }
 }
+
+Cypress.on("uncaught:exception", (err, _runnable) => {
+  return false;
+});
```

Comment:
This disables failing on uncaught exceptions globally (including tests of maas-ui itself) which is not what we want. As a workaround, this should be as narrow as possible (and target maas.io errors that prevent checks from passing).

This can be done by e.g. checking the error message and only returning false for those that are returned from maas.io when testing links.

Alternatively you can provide a stub response for network requests that we do not care about and that seem to be failing. 
https://docs.cypress.io/api/commands/intercept#Controlling-the-response

Reply:
Does this filter make more sense? 
```javascript
Cypress.on("uncaught:exception", (err, _runnable) => {
  // run tests if canonicalGlobalNav, hljs, drpNs is not defined
  if (
    err.message.includes("canonicalGlobalNav") ||
    err.message.includes("hljs") ||
    err.message.includes("drpNs")
  ) {
    return false;
  }
});
```

Reply:
It is an improvement, might be worth explaining in the comment why we're doing this.